iree/tools/CMakeLists.txt

```diff
@@ -32,6 +32,12 @@ iree_select_compiler_opts(IREE_COMPILER_TARGET_BACKENDS
     "iree::compiler::Dialect::HAL::Target::LLVM"
 )
 
+set(IREE_COMPILER_DIALECT_MODULES
```

Comment:
This is going to get overridden by bazel_to_cmake when we get sh_binary working so this file can be processed.